consensus/src/ratification/step.rs

```diff
@@ -4,50 +4,101 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use crate::commons::{spawn_cast_vote, ConsensusError, Database};
+use crate::commons::{ConsensusError, Database, RoundUpdate};
 use crate::config;
 use crate::contract_state::Operations;
 use crate::execution_ctx::ExecutionCtx;
 use std::marker::PhantomData;
 
+use crate::msg_handler::MsgHandler;
 use crate::ratification::handler;
-use node_data::ledger::{to_str, Block};
-use node_data::message::{Message, Payload, Topics};
+use node_data::ledger::to_str;
+use node_data::message;
+use node_data::message::payload::{Ratification, ValidationResult};
+use node_data::message::{AsyncQueue, Message, Payload, Topics};
 use std::sync::Arc;
 use tokio::sync::Mutex;
 
+use tracing::{debug, error};
+
 pub struct RatificationStep<T, DB> {
     handler: Arc<Mutex<handler::RatificationHandler>>,
-    candidate: Option<Block>,
+
     timeout_millis: u64,
-    executor: Arc<Mutex<T>>,
+    _executor: Arc<Mutex<T>>,
 
     marker: PhantomData<DB>,
 }
 
+impl<T: Operations + 'static, DB: Database> RatificationStep<T, DB> {
+    pub async fn cast_vote(
```

Comment:
Why is this different from Validation's `spawn_try_vote`/`try_vote`? 
Can we "unify" the format?

Reply:
Renamed to try_vote as it does not spawn a task.

Reply:
I meant "why do we spawn a task for Validation and not for Ratification"?

Also, I'd rather unify all to `cast_vote` instead of `try_vote`.